src/main/webapp/CreateAnEvent.html

```diff
@@ -56,12 +56,16 @@ <h3>COVID-19 Safety Notice: <br> </h3>
       <br><br>
 
       <div id="attendees-wrap">
-        <label for="attendees">Please enter attendees in a comma separated list:</label><br>
-        <input type="text" id="attendees" name="attendees">
+        <label for="number-attendees">Please enter the number of guests to invite: (max 30)</label>
+        <input type="number" id="number-attendees" name="number-attendees" min="0" max="30">
```

Comment:
Why specifically a max of 30?

Reply:
no specific reason, I was just making sure the implementation would work with a max